suites/reef/rgw/tier-1_rgw_ms_upgrade_5_3GA_to_7x_latest.yaml

```diff
@@ -363,16 +363,11 @@ tests:
 
   - test:
       abort-on-fail: true
-      desc: Multisite upgrade
+      desc: rgw-MS mixed-mode ceph upgrade
```

Comment:
@viduship , could you add details on the site being upgraded in both places, L366 & L422

Reply:
done @mkasturi18